docs/guides/es2k/building-p4cp-acc.md

```diff
@@ -0,0 +1,132 @@
+# Building P4 Control Plane for the ACC
+
+This document explains how to build P4 Control Plane for the ARM Compute
+Complex (ACC) of the Intel&reg; IPU E2100.
+
+## Prepare the System
+
+There are several things to do before you can build P4 Control Plane.
+
+- Install CMake 3.15 or above
+
+  Avoid versions 3.24 and 3.25. They cause the dependencies build to fail.
+
+- Install OpenSSL 1.1
+
+  Note that P4 Control Plane is not compatible with BoringSSL.
+
+- Install the ACC SDK
+
+  See [Installing the ACC SDK](installing-acc-sdk.md) for instructions.
+
+## Install Stratum Dependencies
+
+See [Building Stratum Dependencies for the ACC](../../../setup/building-es2k-stratum-deps.md)
+for instructions.
+
+You will need the Host depencies, Target dependencies, and Build environment
+setup file, in order to cross-compile P4 Control Plane for the ACC.
+
+## Fetch Source Code
```

Comment:
As per the document flow, is it better to 
"Fetch Source Code" section and then "Install Stratum Dependencies" ?

Reply:
I'm treating "Install Stratum Dependencies" as a precursor step. I was originally going to put it in "Prepare the System", but decided that it should be given a little more emphasis.

The plan is to move the Stratum dependencies to a separate repository (not a submodule). I'm kind of paving the way for that.